Lib/test/test_pdb.py

```diff
@@ -907,11 +907,18 @@ def test_post_mortem_chained():
 def test_post_mortem_cause_no_context():
     """Test post mortem traceback debugging of chained exception
 
+    >>> def make_ex_with_stack(type_, *content, from_=None):
```

Comment:
Can we use "exc" instead of "ex" for exception? I think this is more common.

Reply:
done